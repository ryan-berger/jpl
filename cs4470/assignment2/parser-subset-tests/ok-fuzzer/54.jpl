read image "" to a6n.3M_
return G
return .15069
time print "u"
print "mz"
show 86
print ""
show J()
assert J0l_4o_5u_1C_, "y~"
write image N(.77294440) to ""
write image p to ""
