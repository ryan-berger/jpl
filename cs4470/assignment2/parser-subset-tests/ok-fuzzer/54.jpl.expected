(ReadImageCmd "" (VarArgument a6n.3M_))
(StmtCmd (ReturnStmt (VarExpr G)))
(StmtCmd (ReturnStmt (FloatExpr 0)))
(TimeCmd (PrintCmd "u"))
(PrintCmd "mz")
(ShowCmd (IntExpr 86))
(PrintCmd "")
(ShowCmd (CallExpr J))
(StmtCmd (AssertStmt (VarExpr J0l_4o_5u_1C_) "y~"))
(WriteImageCmd (CallExpr N (FloatExpr 0)) "")
(WriteImageCmd (VarExpr p) "")
